src/plugins/intel_cpu/src/nodes/mkldnn_mvn_node.cpp

```diff
@@ -781,6 +834,77 @@ void MKLDNNMVNNode::initSupportedPrimitiveDescriptors() {
     pushDesc(LayoutType::ncsp, impl_type);
 }
 
+MKLDNNMVNNode::MVNExecutor::MVNExecutor(const MVNAttrs& mvnAttrs)
+    : mvnAttrs(mvnAttrs),
+      src_data_size(mvnAttrs.src_prc.size()),
+      dst_data_size(mvnAttrs.dst_prc.size()) {}
+
+MKLDNNMVNNode::MVNJitExecutor::MVNJitExecutor(const MVNAttrs& mvnAttrs,
+                                              const mkldnn::primitive_attr& attr):
+                                              MVNExecutor(mvnAttrs) {
+    auto jcp = jit_mvn_config_params();
+    jcp.src_prc = mvnAttrs.src_prc;
+    jcp.dst_prc = mvnAttrs.dst_prc;
+    jcp.src_data_size = MKLDNNExtensionUtils::sizeOfDataType(MKLDNNExtensionUtils::IEPrecisionToDataType(jcp.src_prc));
+    jcp.dst_data_size = MKLDNNExtensionUtils::sizeOfDataType(MKLDNNExtensionUtils::IEPrecisionToDataType(jcp.dst_prc));
```

Comment:
```suggestion
    jcp.src_data_size = src_data_size;
    jcp.dst_data_size = dst_data_size;
```
as we already created these members in base MVNExecutor.

Reply:
Fixed in PR9889